docs/usage/management-api-walkthrough/02_policies.md

```diff
@@ -28,29 +233,185 @@ Content-Type: application/json
+              "odrl:leftOperand": "BusinessPartnerNumber",
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "{{BPN6789}}"
+            },
+            {
+              "@type": "Constraint",
+              "odrl:leftOperand": "Dismantler",
+              "odrl:operator": {
+                "@id": "odrl:eq"
+              },
+              "odrl:rightOperand": "active"
+            }
+          ]
+        }
+      }
+    ]
+  }
+}
 ```
 
-In the EDC, policies are pure [ODRL (Open Digital Rights Language)](https://www.w3.org/TR/odrl-model/).
-Like the payloads of the [Dataspace Protocol](1-management-api-overview), they are written in JSON-LD. Even if the user
-only has rudimentary knowledge of JSON-LD, the [policy playground](https://eclipse-tractusx.github.io/tutorial-resources/policy-playground/)
-will provide a good starting point to start writing policies. It is important to keep in mind that the extensive ODRL-
-context (that the EDC is aware of) allows for ergonomic reuse of the vocabulary in individual policies.
+##### Desired Outcome (Scenario 3)
+
+Partner 2 does not receive the Contract Offer as he does not own the Dismantler Credential and is therefore not able to 
+negotiate the contract or request the data.
+
+#### Scenario 4
+
+Partner 1 wants to create an Access Policy that Partner 2 can receive the Contract Offer if the BPN is matching. 
+Further, Partner 1 only wants to create an Agreement if Partner 2 is identified as a Dismantler (owns the 
+"DismantlerCredential"). Otherwise the Negotiation has to fail.
+
+##### Partner 1 - Access Policy Example (Scenario 4)
+
+```json
+{
+    "@context": {
+        "odrl": "http://www.w3.org/ns/odrl/2/"
+    },
+    "@type": "PolicyDefinitionRequest",
+    "@id": "{{POLICY_ID}}",
+    "policy": {
+        "@type": "Policy",
+        "odrl:permission": [
+            {
+                "odrl:action": "use",
+                "odrl:constraint": {
+                    "@type": "LogicalConstraint",
+                    "odrl:and": [
+                        {
+                            "@type": "Constraint",
+                            "odrl:leftOperand": "BusinessPartnerNumber",
+                            "odrl:operator": {
+                                "@id": "odrl:eq"
+                            },
+                            "odrl:rightOperand": "{{BPN6789}}"
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+}
+```
+
+##### Partner 1 - Usage/Contract Policy Example (Scenario 4)
+
+```json
+{
+    "@context": {
+        "odrl": "http://www.w3.org/ns/odrl/2/"
+    },
+    "@type": "PolicyDefinitionRequest",
+    "@id": "{{POLICY_ID}}",
+    "policy": {
+        "@type": "Policy",
+        "odrl:permission": [
+            {
+                "odrl:action": "use",
+                "odrl:constraint": {
+                    "@type": "LogicalConstraint",
```

Comment:
we should remove the not required LogicalConstraint in Usage Policies.

Reply:
I said it before and I'll say it again: The implementation can handle logical constraints. If they are discouraged, this should be laid out in normative documents, not in descriptive ones like usage docs.

Reply:
Then they must be able to handle both options. Example her:
https://github.com/eclipse-tractusx/tractusx-profiles/issues/12

Both are ODRL compliant and have the same meaning. This is the thing that comes with ODRL ;-)

Reply:
> we should remove the not required LogicalConstraint in Usage Policies.

To expand on what @arnoweiss said, we will absolutely not eliminate `LogicalConstraint` support from the Tractus-X EDC implementation. If a use case or dataspace member has a need for logical constraints, TX-EDC will be able to support them.

That is distinct from the question of interoperability. A dataspace such as Catena-X may decide the minimum bar for interoperability is that all participants must support an atomic constraint or a logical 'and' constraint. In that case, I'll point out two things:

1. The must-support statement does not restrict systems from implementing more than that. Most interoperability efforts I am familiar with take this approach.
2. Removing `LogicalConstraint` support would remove support for `and` constraints and any multiplicity constraint. This would mean that policies could only be authored with atomic constraints, which would be a severe limitation.

Reply:
Yes, I agree, we should not "remove" it from EDC. that is clear. As long as EDC does also support ODRL allowed way of describing this with:

```
"constraint": [
             {
              "leftOperand": "cx-policy:FrameworkAgreement",
              "operator": "eq",
              "rightOperand": "traceability:v1.0"
            },
            ...
        ]
```
all is good and I only would like to see this in the documentation, that users are aware they need to be able to handle both. Otherwise, we would need to restrict how ODRL can be used.
@jimmarino  Do we have any issue with EDC with the given example? When I tried, I couldn't' detect one with 0.5.3

Reply:
Ok I'm giving up on this. Taking this as tx-edc setting the rules here. It must be `LogicalConstraint->and` , independent from what may be allowed with ODRL.

Reply:
That's not what I said. As I indicated, TX-EDC will continue to support multiple logical constraint types, including specifying them in ODRL serialized as Json-Ld. It is also reasonable for upstream EDC to support not specifying `@type` and interpreting that as a logical `and` constraint. However, if that is not supported, it should be submitted as a feature request to the EDC project.

This is how Eclipse open-source projects work. It's not about "tx-edc setting the rules." It's about providing proper governance overseen by the committers.

Reply:
> This is how Eclipse open-source projects work. It's not about "tx-edc setting the rules." It's about providing proper governance overseen by the committers.

yes, correct. not the best wording I used. Thanks for the clarification @jimmarino